Inventory System with Directus Data Studio

Victory-ET opened the pull request:
First draft submission for the article on: Building an Inventory Management System with Directus Data Studio

Victory-ET committed:
Inventory System with Directus Data Studio

phazonoverload commented:
This isn't really what I expected to see - this is just a basic tutorial on adding and sorting data to a single collection in the Data Studio. 

I would expect to see multiple collections - perhaps `products`, `inventory`, and possibly `orders`. Automations to keep everything in sync as a true inventory management system. Your proposal also mentions `Providing features for generating reports and analyzing inventory data to make informed business decisions.` which are missing from this submission. 

Unfortunately, I can't accept this in it's current form as it doesn't contribute anything new to the body of knowledge that we have published. 

If you can turn around a suitable amendment by the end of Friday then I can squeeze time in to review and publish it. Otherwise, we won't be able to accept this submission.

Victory-ET committed:
update to article

Victory-ET commented:
@phazonoverload 

My sincere apologies for the article coming up short of what was expected. I have modified the draft to feature more operations of an inventory system, and updated the PR. Thank you for the extension. I used panels from the dashboard to visualize data and produce reports based on certain conditions. While this is slightly different from the type of report I had envisioned while writing the proposed outline (due to the flexibility of JavaScript for data manipulation), I have tried to replicate same using the Data Studio.

phazonoverload commented:
Thank you for writing this, and we are happy to consider it done in it's current state. There are some edits required which my team will do when we have time, so I can't offer an estimate timeline for publishing.

This repo is just for authoring/reviewing - the actual piece will be published via a Directus project, so we'll take it from here. I'll close this issue to keep the main branch clean, but this is a acceptance of your work.

Could you email us at [devrel@directus.io](mailto:devrel@directus.io) and we'll share what we need to get you paid. Thanks!
